splunk/scripts/log_indexing.sh

```diff
@@ -5,9 +5,25 @@ endpoint="https://localhost:8089/services/data/inputs/oneshot"
 SPLUNK_PASSWORD="Password"
 
 echo "Indexing logs..."
+max_retries=3
+
 for file in "$logs_path"/*; do
   echo "Indexing $file"
-  curl  --insecure --user admin:"$SPLUNK_PASSWORD" "$endpoint" --data name="$file" --data index="test_index"
+  retries=0
+
+  while [ $retries -lt $max_retries ]; do
```

Comment:
please read up on the difference between the single and double-bracket syntax in bash, and try to use the newer syntax where possible.

Alternatively, we might be able to user `curl`s built-in `--retry` option.

Reply:
nice one, fixed